### ISSUE TITLE
awsnycast should initiate failover when route is active but target instance is stopping

### ISSUE THREAD

**Issue by somic** [bobtfish/AWSnycast#21]
If I understand correctly, awsnycast will repoint a route to another instance only when the route says it's blackholed.

I am still looking for evidence (in internal logs and around the Internet) but it looks to me like aws will mark a route "blackholed" only when its target instance is in "terminated" or "stopped" states.

To me it looks like a bug on their end - they should also blackhole the route when its target instance is in "stopping" state (for more details, please see internal DAR-233).

Thoughts about this? Also, can you please point me to a place in the codebase where this logic is implemented? I tried grepping for blackhole but didn't find anything - will appreciate a hint.

**Comment by bobtfish** [comment]
I agree they *should* do blackhole as soon as stopping/shutting-down, but they don't. Maybe we should inspect the instance state and look for this as well..

https://github.com/bobtfish/AWSnycast/blob/master/aws/route_table_manager.go#L227 is probably what you're looking for code wise :)
